packages/__docs__/tsconfig.build.json

```diff
@@ -273,6 +273,9 @@
     {
       "path": "../ui-babel-preset/tsconfig.build.json"
     },
+    {
+      "path": "../ui-color-picker/tsconfig.build.json"
+    },
```

Comment:
This will type check the docs app during build, it just increases build time by ~1 sec